dellyGermline.wdl

```diff
@@ -162,10 +222,10 @@ task genotype{
 
   parameter_meta {
     mergedBcf: "Merged bcf file from >=20 unrelated samples"
-    inputBam: "bam file to be genotyped"
-    inputBamIndex: "index file for bam that will be genotyped"
-    outputFileNamePrefix: "sample ID, this is provided to maivs and cannot include reseerved characters [;,_\\s] "
-    modules: "modules needed to run Delly"
+    inputBam: "Bam file to be genotyped"
+    inputBamIndex: "Index file for bam that will be genotyped"
+    outputFileNamePrefix: "Sample ID, this is provided to Mavis and cannot include reserved characters [;,_\\s] "
```

Comment:
please remove any references to mavis, as it is not relevant for this workflow. can use `outputFileNamePrefix: "sample ID"`